redmine_zulip/lib/zulip_hooks.rb

```diff
@@ -105,10 +105,7 @@ def zulip_port()
    end
 
    def zulip_api_basename()
-     if Setting.plugin_redmine_zulip[:zulip_server]["api.zulip.com"]
-       return ""
-     end
-     return "/api"
+     return (Setting.plugin_redmine_zulip[:zulip_server].sub %r{^https?:(//|\\\\)(www\.)?}i, '').concat('/api')
```

Comment:
OK, makes sense; let's drop the `www` piece (I don't think anyone will be typing `www` URLs) but leave the http(s) stuff.